satpy/readers/seviri_l1b_native.py

```diff
@@ -370,13 +370,16 @@ def _read_header(self):
         self.mda['hrv_number_of_lines'] = int(sec15hd["NumberLinesHRV"]['Value'])
         self.mda['hrv_number_of_columns'] = cols_hrv
 
-        if self.header['15_MAIN_PRODUCT_HEADER']['QQOV']['Value'] == 'NOK':
-            warnings.warn(
-                "The quality flag for this file indicates not OK. "
-                "Use this data with caution!",
-                UserWarning,
-                stacklevel=2
-            )
+        if '15_MAIN_PRODUCT_HEADER' not in self.header:
+            logger.info("Quality flag check was not possible due to missing 15_MAIN_PRODUCT_HEADER.")
```

Comment:
info or debug? Put another way, is this something you want to see most SIFT users seeing in an INFO-level log?

Reply:
seems to me that info is more suited here, since it's not a debug message (useful for troubleshooting or so), it's just informing about a state of the code, and no futher action is needed in most cases, nor done in the code. But if there are stronger guidelines on this, I can change it of course.